src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -492,7 +498,23 @@ public static String injectContextDependentValuesForTest(String content) {
                                StringHelper.truncateLongId(TestProperties.TEST_ADMIN_ACCOUNT))
                       .replace("<!-- nexthour.date -->", TimeHelper.formatDate(TimeHelper.getNextHour()))
                       .replace("<!-- now.datetime -->", TimeHelper.formatTime12H(now))
-                      .replace("<!-- now.datetime.comments -->", TimeHelper.formatDateTimeForComments(now));
+                      .replace("<!-- now.datetime.comments -->", TimeHelper.formatDateTimeForComments(now, 0));
+    }
+
+    private static TimeZone getTimeZone(String content) {
+        //Searches for last String of pattern "UTC+0800" in the content.
```

Comment:
This comment does not seem relevant. You should not be searching for "UTC+8000" in particular.